### PR TITLE
mavenization

### PR DESCRIPTION
**Pull request by senivam**
Signed-off-by: Maxim Nesen <maxim.nesen@oracle.com>

### CODE REVIEW

File: jaxrpc-ri/pom.xml [hunk 1]
```diff
@@ -13,90 +13,182 @@
...
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <version>4.12</version>
-            <scope>test</scope>
-        </dependency>
-    </dependencies>
+    <organization>
+        <name>Eclipse Foundation</name>
+        <url>https://www.eclipse.org</url>
+    </organization>
```
Comment (lukasj) [inline jaxrpc-ri/pom.xml]: remove, let it be inherited from parent

File: jaxrpc-ri/pom.xml [hunk 2]
```diff
@@ -13,90 +13,182 @@
...
+
+    <properties>
+        <mail.api.version>1.6.3</mail.api.version>
+        <activation.api.version>1.2.1</activation.api.version>
+        <servlet.api.version>4.0.2</servlet.api.version>
+        <rpc.api.version>1.1.3</rpc.api.version>
+        <soap.api.version>1.4.1</soap.api.version>
+        <saaj.version>1.5.1</saaj.version>
+        <xerces.version>2.6.2-jaxb-1.0.6</xerces.version>
+        <fastInfoset.version>1.2.15</fastInfoset.version>
```
Comment (lukasj) [inline jaxrpc-ri/pom.xml]: 1.2.16 is the jakarta provided version - check with @Tomas-Kraus to see it's ready

File: jaxrpc-ri/pom.xml [hunk 3]
```diff
@@ -13,90 +13,182 @@
...
+                    <version>3.8.0</version>
+                    <configuration>
+                        <release>6</release>
+                    </configuration>
+                </plugin>
+                <plugin>
+                    <groupId>org.apache.felix</groupId>
+                    <artifactId>maven-bundle-plugin</artifactId>
+                    <version>3.5.1</version>
+                    <executions>
```
Comment (lukasj) [inline jaxrpc-ri/pom.xml]: keep only configuration element here and move the rest to the plugins/plugin section

File: jaxrpc-ri/pom.xml [hunk 4]
```diff
@@ -13,90 +13,182 @@
...
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
     </build>
+
+    <modules>
+        <module>spi</module>
+        <module>impl</module>
+        <module>assembly</module>
```
Comment (lukasj) [inline jaxrpc-ri/pom.xml]: I'd prefer this to be called distribution or bundle but not a big deal...

File: jaxrpc-ri/spi/pom.xml [hunk 5]
```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2018 Payara Foundation and/or its affiliates.
```
Comment (lukasj) [inline jaxrpc-ri/spi/pom.xml]: I don't think you are Payara employee. Or just not yet? :-)

**Comment by senivam** [comment]
all notes except *FastInfoset* version fixed. 

Version will be checked

**Comment by senivam** [comment]
All notes resolved
*FastInfoset* version updated

**Pull request closed without merging**